### PR TITLE
RgbColor: Add support for color titles

### LINKED ISSUE #3572: RGB Color: Add names to colors
The names are already available in the response (for many, many colors), so when available we should display the color name.

It'd be nice to have the names for the input colors, but we may be a bit pressed for space.

------
IA Page:  http://duck.co/ia/view/rgb_color 
[Maintainer](http://docs.duckduckhack.com/maintaining/guidelines.html): @GuiltyDolphin

### PR DESCRIPTION
**Pull request by GuiltyDolphin**
This adds support for displaying the color name/title on hover over the hex/circle respectively.

Resolves #3572 (at least in part - I find this satisfactory for now).

cc @mintsoft 

---

Instant Answer Page: https://duck.co/ia/view/rgb_color

[Maintainer](http://docs.duckduckhack.com/maintaining/guidelines.html): @GuiltyDolphin

### CODE REVIEW

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

File: share/goodie/rgb_color/content.handlebars [hunk 1]
```diff
@@ -1,6 +1,6 @@
-<a href='{{colorPickerLink result_color.hex}}' class='rgb_color--box circle' style='background:#{{result_color.hex}}'/>
+<a title="{{result_color.title}}" href='{{colorPickerLink result_color.hex}}' class='rgb_color--box circle' style='background:#{{result_color.hex}}'/>
```
Comment (mintsoft) [inline share/goodie/rgb_color/content.handlebars]: Hmm personally I prefer `<a href='' title=''/>` but whatever floats your boat

**Comment by mintsoft** [comment]
@moollaza can we check these out on beta?

File: share/goodie/rgb_color/content.handlebars (hunk 1 above)
Comment (GuiltyDolphin) [inline share/goodie/rgb_color/content.handlebars]: My boat doth float.

**Comment by moollaza** [comment]
@GuiltyDolphin Thanks! This LGTM :+1:

**Pull request merged**